graphstorm-processing/graphstorm_processing/config/config_conversion/gconstruct_converter.py

```
@@ -0,0 +1,189 @@
+                label_type = label["task_type"]
+                label_dict = {"column": label_column, "type": label_type}
+                if "split_pct" in label:
+                    label_splitrate = label["split_pct"]
+                    # check if split_pct is valid
+                    assert sum(label_splitrate) == 1.0, "sum of the label split rate should be 1.0"
+                    label_dict["split_rate"] = {
+                        "train": label_splitrate[0],
+                        "val": label_splitrate[1],
+                        "test": label_splitrate[2],
+                    }
+                if "separator" in label:
+                    label_sep = label["separator"]
+                    label_dict["separator"] = label_sep
+                labels_list.append(label_dict)
+            except KeyError as exc:
+                raise KeyError(f"A required key was missing from label input {label}") from exc
+        return labels_list
+
+    @staticmethod
+    def _convert_feature(feats: list[dict]) -> list[dict]:
+        """Convert the feature config
+        Parameters
+        ----------
+        feats: list[dict]
+            The feature information in the GConstruct format
+
+        Returns
+        -------
+        list[dict]
+            The feature information in the GSProcessing format
+        """
+        feats_list = []
+        if feats in [[], [{}]]:
+            return []
+        for ele in feats:
+            if "transform" in ele:
+                raise ValueError(
+                    "Currently only support no-op operation, "
+                    "we do not support any other no-op operation"
+                )
+            feat_dict = {}
+            kwargs = {"name": "no-op"}
+            for col in ele["feature_col"]:
+                feat_dict = {"column": col, "transform": kwargs}
+                feats_list.append(feat_dict)
+            # TODO: out_dtype: currently gsprocessing only support one type
```

Can we also raise an error when out_dtype is not float32?